.github/workflows/build.yml

```diff
@@ -97,39 +97,34 @@ jobs:
           retention-days: ${{ env.RETENTION }}
 
 
-  # build-linux:
-  #   strategy:
-  #     fail-fast: false
-  #     matrix:
-  #       version: ["4.27", "5.2"]
-  #   runs-on: ["self-hosted", "X64", "Windows"]
-  #   env:
-  #     BUILD_PLATFORM: 'linux64'
-  #   steps:
-  #     - name: Code checkout
-  #       uses: actions/checkout@v3
-  #     - name: Download C++ SDK release asset for Windows
-  #       id: download_cpp_win
-  #       uses: ./.github/actions/download_cpp/powershell
-  #       with:
-  #         url: ${{ env.CPP_SDK_URL }}
-  #         tag: ${{ env.CPP_SDK_TAG }}
-  #         asset_name: "cppsdk-${{ env.CPP_SDK_TAG }}-windows64.zip"
-  #     - name: Download C++ SDK release asset
-  #       id: download_cpp
-  #       uses: ./.github/actions/download_cpp/powershell
-  #       with:
-  #         url: ${{ env.CPP_SDK_URL }}
-  #         tag: ${{ env.CPP_SDK_TAG }}
-  #         asset_name: "cppsdk-${{ env.CPP_SDK_TAG }}-${{ env.BUILD_PLATFORM }}.zip"
-  #     - name: Run Build
-  #       run: |
-  #         Write-Output "UE_${{ matrix.version }}"
-  #         $UERunUAT = "C:\\Program Files\\Epic Games\\UE_${{ matrix.version }}\\Engine\\Build\\BatchFiles\\RunUAT.bat"
-  #         & $UERunUAT BuildPlugin -Rocket -Plugin="$(pwd)\\DolbyIO\\DolbyIO.uplugin" -TargetPlatforms=Linux -Package="$(pwd)\\Build\\DolbyIO" -StrictIncludes -VS2019 -NoTurnKeyVariables
-  #     - name: Archive DolbyIO Plugin
-  #       uses: actions/upload-artifact@v3
-  #       with:
-  #         name: DolbyIO_UnrealPlugin_${{ matrix.version }}_${{ env.BUILD_PLATFORM }}
-  #         path: ${{ env.UNREAL_PLUGIN_OUTPUT }}
-  #         retention-days: ${{ env.RETENTION }}
+  build-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ["5.2"]
+    runs-on: ["self-hosted", "X64", "Windows"]
+    env:
+      BUILD_PLATFORM: 'linux64'
+      LINUX_MULTIARCH_ROOT: "C:\\UnrealToolchains\\v21_clang-15.0.1-centos7"
```

Comment:
@msobo1 @pczaj @jsobi Is it possible for LINUX_MULTIARCH_ROOT to depend on matrix.version? I would like to acheive something like:
```
if matrix.version == 4.27:
    LINUX_MULTIARCH_ROOT: "C:\\UnrealToolchains\\v19_clang-11.0.1-centos7"
else:
    LINUX_MULTIARCH_ROOT: "C:\\UnrealToolchains\\v21_clang-15.0.1-centos7"

Reply:
LINUX_MULTIARCH_ROOT would need to be removed from "env" section and set at the beginning of the "run" section:
`if(${{ matrix.version }} -eq "4.27"){
   set LINUX_MULTIARCH_ROOT="C:\\UnrealToolchains\\v19_clang-11.0.1-centos7"
}else {
   set LINUX_MULTIARCH_ROOT="C:\\UnrealToolchains\\v21_clang-15.0.1-centos7"
}`

Reply:
Thanks. I needed to do $env:LINUX_MULTIARCH_ROOT instead of set but otherwise it works.